### DISCUSSION TITLE
Welcome to jotai-devtools Discussions!

### DISCUSSION THREAD

**Discussion by arjunvegda** [jotaijs/jotai-devtools#1] [Announcements]

**Comment by hamid842** [comment]
Hello
is it need to install DevTools as devdependencies?

**Reply by arjunvegda** [reply]
Thanks for reaching out @hamid842!

That's totally up to you! I declare it as a dependency on my projects. Also, `<DevTools/>` is designed to be tree-shaked in a production build you don't have to worry about it bloating up the production bundle size 🙌

**Reply by cristianmaridev** [reply]
onChange para <Devtools /> como guardo el ultimo estado del componente sino xD
